packages/eds-core-react/src/components/Input/Input.docs.mdx

```diff
@@ -0,0 +1,53 @@
+import { Links, PropsTable, Story } from './../../../.storybook/components'
+import { Input } from '.'
+
+# Input
+
+Is intended to use if you need to more flexibility than the wrapped TextField or select components give.
+
+<Links 
+    ariaUrl="https://www.w3.org/TR/wai-aria-1.1/#input"
+    sourceUrl="https://github.com/equinor/design-system/blob/develop/packages/eds-core-react/src/components/Input/Input.tsx"
+    npmUrl="https://www.npmjs.com/package/@equinor/eds-core-react"
+/>
+
+<Story id="inputs-input--introduction" />
+<PropsTable />
+
+## Usage
+
+```tsx
+import { Input } from '@equinor/eds-core-react'
+
+<Input />
+```
+
+## Examples
+
+### Types
+
```

Comment:
Should it be pointed out that this is the native type prop and here is a small selection of examples of all the possible types?

Reply:
Added

Reply:
"Native `type` property" (not "native type of property")